HPXMLtoOpenStudio/resources/hvac.rb

```diff
@@ -3077,6 +3077,12 @@ def self.set_heat_c_d(heating_system, num_speeds)
     htg_ap.heat_plf_fplr_spec = [calc_plr_coefficients(htg_ap.heat_c_d)] * num_speeds
   end
 
+  def self.set_cool_ceer(cooling_system)
+    return if cooling_system.cooling_system_type != HPXML::HVACTypeRoomAirConditioner
+    # eer must be specified by EPvalidator.
+    cooling_system.cooling_efficiency_ceer = cooling_system.cooling_efficiency_eer / 1.01
```

Comment:
Let's add a comment citing the ENERGY STAR document that this regression is based on.